dist/s_all

```diff
@@ -47,6 +47,12 @@ errchk()
 		return
 	fi
 
+	# Return if evergreen validate runs sucessfully.
+	if echo "$1" | grep -q evergreen && [ "$(cat $2)" = "../test/evergreen.yml is valid" ] ; then
```

Comment:
Missing quotes around `$2`, see [SC2086]( https://github.com/koalaman/shellcheck/wiki/SC2086)

Reply:
but isn't $2 a file name? (Just one word?)

Reply:
It's about preventing globbing and word splitting